x/bep3/client/cli/query.go

```diff
@@ -26,6 +26,40 @@ const (
 	flagDirection  = "direction"
 )
 
+type FormattedAtomicSwap struct {
```

Comment:
Should this go in `x/bep3/types/swap.go`?  Is there a better existing pattern for decorating cli output?

Reply:
Agree, it seemed out of place outside of client.  The naming is tough.

Updating the cli and rest client to match will let me push the new interface into the `quierier`, which I think solves some of the ugliness of declaring the type in the client.

AugmentedAtomicSwap sounds better in that context, and likely better to stick with a single pattern.  Thanks for the example